src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+
+const tasks = [
+  {
+    index: 1,
+    description: 'structure todo list',
+    completed: false,
+  },
+  {
+    index: 2,
+    description: 'add task todo list',
+    completed: false,
+  },
+  {
+    index: 3,
+    description: 'remove todo list',
+    completed: false,
+  },
+];
+
+function populateList() {
+  const todoList = document.getElementById('todo-list');
+
+  tasks.forEach((item) => {
+    const listItem = document.createElement('li');
+    listItem.innerHTML = `
+      <input class="checkbox" type="checkbox" name="" value="">
+      <span>${item.description}</span>
+      `;
+    todoList.appendChild(listItem);
+  });
```

Comment:
-  [ ]. I am not able to add an item please fix that and this prevents me from check the other functionalities of the project so the next reviewer will have right to check them as well



https://user-images.githubusercontent.com/75126481/222556034-21e7b879-ae9c-4825-aeb4-86c6ff17c4da.mov

Reply:
@hiromataba, I think it is nex stage of the project that one can add all interactiveness and functionality required, please check the requirement well. Thanks

Reply:
@hiromataba, Thanks, I got a better understand👌👍❤️

Reply:
Glad to hear that 🙂 @david30-maker